tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-card.html

```diff
@@ -317,6 +329,13 @@
         this._attached = true;
         this.reload();
       },
+      ready() {
+        this.$$('vz-line-chart').addEventListener('chart-rendered', () => {
+          // Reset the chart's X and Y domains to [0, 1] only after the chart is
+          // rendered. Otherwise, the call to resetDomain no-ops. 
+          this.$$('vz-line-chart').resetDomain();
```

Comment:
Ok, I see why we needed the customevent above. This seems a little hacky to me. Why not modify the Plottable code so that when the scales are initialized, if a default range was provided, the default range is used?

Reply:
Like this? See the `renderTo` method. Done.